_posts/2019-07-08-Cypress.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: post
+authors: [tim_vierbergen,glenn_van_de_putte]
+title: 'Cypress: a new kid on the E2E block'
+image: /img/cypress/cypress-logo-black.png
+tags: [ci,e2e,javascript,testing,selenium,cypress]
```

Comment:
Use capital letters and check tags which are already there (Continuous Integration, JavaScript, Testing, Selenium, Cypress)